community/modules/compute/gke-job-template/README.md

```diff
@@ -0,0 +1,98 @@
+## Description
+
+This module is used to create a Kubernetes job template file.
+
+The job template file can be submitted as is or used as a template for further
+customization. Add the `instructions` output to a blueprint (as shown below) to
+get instructions on how to use `kubectl` to submit the job.
+
+This module is designed to`use` a `gke-node-pool` module.
```

Comment:
In GKE, can you submit a job to multiple node pools? (this could make sense where pools are in multiple zones or regions). In that case, can you use multiple?

Reply:
It seems like core GKE functionality and the module has been updated to support this.